test/vars/UtilSpec.groovy

```diff
@@ -464,9 +466,11 @@ class UtilSpec extends JenkinsPipelineSpecification {
         groovyScript.storeGitInformation('projectName')
         then:
         1 * getPipelineMock('githubscm.getCommit')() >> 'kiegroup/lienzo-core: 0f917d4 Expose zoom and pan filters (#102)'
+        1 * getPipelineMock('githubscm.getCommitHash')() >> '11111111111111111111111111111111'
         1 * getPipelineMock('githubscm.getBranch')() >> '* (detached from 0f917d4)  remotes/origin/master'
         1 * getPipelineMock('githubscm.getRemoteInfo')('origin', 'url') >> 'https://github.com/kiegroup/lienzo-core.git'
         env['GIT_INFORMATION_REPORT'] == 'projectName=kiegroup/lienzo-tests: 45c16e1 Fix tests (#84) Branch [* (detached from 45c16e1)  remotes/origin/master] Remote [https://github.com/kiegroup/lienzo-tests.git]; projectName=kiegroup/lienzo-core: 0f917d4 Expose zoom and pan filters (#102) Branch [* (detached from 0f917d4)  remotes/origin/master] Remote [https://github.com/kiegroup/lienzo-core.git]'
+        env['GIT_INFORMATION_HASHES'] == "projectName=11111111111111111111111111111111"
```

Comment:
I guess here you should test with a previous value stored in `env['GIT_INFORMATION_HASHES']` as it is done for `env['GIT_INFORMATION_REPORT']`. wdyt ?

Reply:
you are totally right, done!